### PR TITLE
Fix a few QtGui -> QtWidget issues

### PR DESCRIPTION
**Pull request by hengestone**
Possibly caused by `pyqtgraph`

### CODE REVIEW

**Comment by hengestone** [comment]
Will update this later

**Comment by goodboy** [comment]
@hengestone give this a rebase onto `master` if you don't mind. #209 should let the CI run now?

File: piker/ui/_chart.py
Comment (goodboy) [inline piker/ui/_chart.py:22]: in 3.9 I'm pretty sure we don't need this?

**Review by goodboy — COMMENTED** [review]
Yeah other then not needing the `typing.List` import any more for python 3.9 (which I think we've now pinned to) this looks good to me 😎

**Comment by goodboy** [comment]
Nice it worked.

File: piker/ui/_chart.py
Comment (hengestone) [reply piker/ui/_chart.py]: OK, removed in `afafe56` below.

**Review by goodboy — APPROVED** [review]
@hengestone just waiting on feedback from one user who needs this same patch to get up and running.

Once we get the ok on that I'll merge this 🏄🏼

**Pull request merged**
